### PR TITLE
update: concept for OS CCR

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
## Describe your changes
The request was to remove HA and DR content from the concept for now. 
Removed the following points: 

- High availability and Disaster recovery (active-passive): With tolerance for outages or complete failure of clusters, cross-cluster replication ensures uninterrupted service availability with the ability to failover to an alternate cluster.

- Centralized reporting cluster: Having a single replica cluster as a single source of truth between different master cluster

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

**Review by hoang-vo — APPROVED** [review]

File: docs/products/opensearch/concepts/cross-cluster-replication-opensearch.md [hunk 1]
```diff
@@ -1,60 +1,39 @@
...
-[limited availability feature](/docs/platform/concepts/beta_services). If you're interested in trying out this feature, contact
-the sales team at [sales@aiven.io](mailto:sales@aiven.io).
-:::
+Cross-cluster replication (CCR) in Aiven for OpenSearch allows you to replicate an entire cluster, including all indexes, mappings, and metadata, from one service to another across different regions and cloud providers.
 
-Cross-cluster replication in Aiven for OpenSearch allows you to
-replicate the entire cluster, including all of its indexes, mappings,
-and metadata, from one service to another across different regions and
-cloud providers.
+This an `active-passive` model, where the follower service pulls all data and
```
Comment (staceysalamon-aiven) [inline docs/products/opensearch/concepts/cross-cluster-replication-opensearch.md:8]: I think this is missing an "is"?

File: docs/products/opensearch/howto/setup-cross-cluster-replication-opensearch.md [hunk 2]
```diff
@@ -1,27 +1,20 @@
 ---
-title: Setup cross cluster replication for Aiven for OpenSearch®
+title: Setup cross-cluster replication for Aiven for OpenSearch®
```
Comment (staceysalamon-aiven) [inline docs/products/opensearch/howto/setup-cross-cluster-replication-opensearch.md:2]:
```suggestion
title: Set up cross-cluster replication for Aiven for OpenSearch®
```

**Review by staceysalamon-aiven — APPROVED** [review]
Just a couple of typos, otherwise LGTM!

File: docs/products/opensearch/concepts/cross-cluster-replication-opensearch.md (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/opensearch/concepts/cross-cluster-replication-opensearch.md:8]: Fixed

File: docs/products/opensearch/howto/setup-cross-cluster-replication-opensearch.md (hunk 2 above)
Comment (harshini-rangaswamy) [reply docs/products/opensearch/howto/setup-cross-cluster-replication-opensearch.md:2]: fixed

**Pull request merged**